### PR TITLE
Add safe-test for asymmetric key pair

### PR DESCRIPTION
**Pull request by whh39**
Add safe-test for asymmetric key pair

the safe-test will support for:
rsa: generate key pair
ecc: generate key pair
sm2: generate key pair

test: passed all the unit test
Signed-off-by: wanghaiheng <haihengx.wang@intel.com>

### CODE REVIEW

File: core/Enclave/key_factory.cpp [hunk 1]
```diff
@@ -261,6 +291,13 @@ sgx_status_t ehsm_create_rsa_key(ehsm_keyblob_t *cmk)
     if (rsa_keypair == NULL)
         goto out;
 
+#ifdef FIPS_MODE
```
Comment (yang8621) [inline core/Enclave/key_factory.cpp:294]: is FIPS_MODE defined?

File: core/Enclave/key_factory.cpp [hunk 2]
```diff
@@ -217,6 +217,36 @@ sgx_status_t ehsm_create_aes_key(ehsm_keyblob_t *cmk)
     return ret;
 }
 
+#ifdef FIPS_MODE
+static bool test_RSA_keypair(RSA *keypair)
+{
```
Comment (syan10) [inline core/Enclave/key_factory.cpp:222]: test_RSA_keypair -> pair_wise_test_for_rsa

FIPS_MODE -> ENABLE_PAIR_WISE_TEST

File: core/Enclave/key_factory.cpp (hunk 1 above)
Comment (whh39) [reply core/Enclave/key_factory.cpp:294]: in makefile

File: core/Enclave/key_factory.cpp (hunk 2 above)
Comment (whh39) [reply core/Enclave/key_factory.cpp:222]: ok

File: core/Enclave/key_factory.cpp (hunk 1 above)
Comment (whh39) [reply core/Enclave/key_factory.cpp:294]: it's a mistake, FIPS_MODE will remove

**Pull request closed without merging**